docs/core/tools/global-json.md

```diff
@@ -1,32 +1,96 @@
+[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
 
-.NET Core CLI tools look for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+The *global.json* file allows you to define which .NET Core SDK version is used when you run .NET Core CLI commands.
+
+.NET Core SDK looks for a *global.json* file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+
+## global.json schema
+
+### sdk
 
-## sdk
 Type: Object
 
-Specifies information about the SDK.
+Specifies information about the .NET Core SDK to select.
+
+#### version
 
-### version
 Type: String
 
-The version of the SDK to use.
+The version of the .NET Core SDK to use.
 
-For example:
+Note that this field:
+
+- Doesn't have globbing support, that is, the full version number has to be specified.
+- Doesn't support version ranges.
+
+The following example shows the contents of a *global.json* file:
 
 ```json
 {
   "sdk": {
-    "version": "1.0.0-preview2-003121"
+    "version": "2.1.300"
   }
 }
 ```
+
+## global.json and the .NET Core CLI
+
+It's helpful to know which versions are available in order to set one in the *global.json* file. You can find the full list of supported available SDKs at the [.NET Downloads](https://www.microsoft.com/net/download/all) site. Starting with .NET Core SDK 2.1, you can run the following command to verify which SDK versions are already installed on your machine:
+
+```console
+dotnet --list-sdks
+```
+
+To install additional .NET Core SDK versions on your machine, visit the [.NET Downloads](https://www.microsoft.com/net/download/all) site.
+
+You can create a new the *global.json* file in the current directory by executing the [dotnet new](dotnet-new.md) command, similar to the following example:
+
+```console
+dotnet new globaljson --sdk-version 2.1.300
+```
+
+## Matching rules
+
+Starting with .NET Core SDK 2.0, the following rules apply when determining which version of the SDK to use:
+
+- If no *global.json* file is found or *global.json* doesn't specify an SDK version, the latest installed SDK version is used. Latest SDK version can be either release or pre-release - the highest version number wins.
+- If *global.json* does specify an SDK version:
+  - If the specified SDK version is found on the machine, that exact version is used.
+  - If the specified SDK version can't be found on the machine, the latest installed SDK **patch-version** of that version is used. Latest installed SDK **patch-version** can be either release or pre-release - the highest version number wins.
+  - If the specified SDK version and an appropriate SDK **patch-version** can't be found, an error is thrown.
+
+The SDK version is currently composed of the following parts:
+
+`[major].[minor].[xyz][-optional preview name]`
```

Comment:
It may be pedantic, but this isn't correct, in the sense I think people will expect. "minor" is not an SDK minor release and there's some indication in this that we use semantic versioning, which we do not. I suggest the following

`[.NET Core major].[.NET Core minor].[xyz][-optional preview name]`

The **feature release** of the .NET Core SDK is represented by the first digit (`x`) in the last portion of the number (`xyz`) for SDK versions 2.1.100 and greater. In general the .NET Core SDK has a faster release cycle than .NET Core.